packages/lesswrong/server/serverStartup.ts

```diff
@@ -105,62 +125,51 @@ async function serverStartup() {
       collection.startPolling();
     }
   }
+}
 
+const runServer = async ({shellMode, command}: CommandLineArguments) => {
```

Comment:
There's also a file called runServer, which calls serverStartup, which calls runServer, which is slightly confusing